opencensus/pom.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2018 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<!-- [START opencensus_pom] -->
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>jar</packaging>
+  <groupId>com.example</groupId>
+  <artifactId>opencensus-samples</artifactId>
+  <version>1.0</version>
+
+  <!--
+    The parent pom defines common style checks and testing strategies for our samples.
+    Removing or replacing it should not affect the execution of the samples in anyway.
+  -->
+  <parent>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.0.10</version>
+  </parent>
+
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
+    <!-- [START dependencies] -->
```

Comment:
This should probably start with `monitoring_opencensus` for the sample tracker

Reply:
I don't see that in the sample tracker. Only one I see is [this one](https://devrel-prod.googleplex.com/samples/1221?product=monitoring), which tracks `monitoring_opencensus_metrics_quickstart`

Reply:
According to the samples rubric, each region tag is supposed to be globally unique and begin with the product name (which seems to be `monitoring`). Maybe `monitoring_opencensus_metrics_dependencies` or `monitoring_opencensus_metrics_quickstart_dependencies` to avoid collisions?

Reply:
Ah I see, so it's not as much for the sample tracker dashboard, it's that the rubric is different from what we've been doing. Will do.